.travis.yml

```diff
@@ -7,29 +7,38 @@ before_install:
   - sudo apt-get install -y libsqlite3-dev iperf3 python3-pip bridge-utils wireguard linux-source linux-headers-$(uname -r) curl git libssl-dev pkg-config build-essential ipset
   - which diesel || cargo install diesel_cli --no-default-features --features sqlite
   - sudo cp $(which diesel) /usr/bin
+  - command -v cross 1>/dev/null || cargo install cross
```

Comment:
so we're ignoring the stdout of this command? won't travis dot hat for us since it's part of the pre setup?

Reply:
This is a trick to check if cross is already installed and if its not installed, it will be. Travis caches ~/.cargo dir, so it might happen that cross binary already exists